src/public/CNAME

```diff
@@ -1 +0,0 @@
-www.ngx-translate.com
```

Comment:
don't delete this one, it's used by surge

Reply:
Just re-added surge and travis

Reply:
ok nice